transactions/src/main/java/org/ehcache/transactions/xa/txmgr/provider/FromConfigurationTransactionManagerProviderConfiguration.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright Terracotta, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.ehcache.transactions.xa.txmgr.provider;
+
+import org.ehcache.spi.service.ServiceCreationConfiguration;
+import org.ehcache.transactions.xa.txmgr.TransactionManagerWrapper;
+
+/**
+ * {@link ServiceCreationConfiguration} for the
+ * {@link FromConfigurationTransactionManagerProvider}.
+ */
+public interface FromConfigurationTransactionManagerProviderConfiguration extends ServiceCreationConfiguration<TransactionManagerProvider> {
```

Comment:
Let's be honest, this is not my favourite class name. Suggestions for change more than welcome 😉